sdk/servicebus/azure-servicebus/azure/servicebus/amqp/_amqp_message.py

```diff
@@ -0,0 +1,554 @@
+        """Compare objects by comparing all attributes."""
+        return not self.__eq__(other)
+
+    def __str__(self):
+        # type: () -> str
+        return str({k: v for k, v in self.__dict__.items() if not k.startswith("_")})
+
+    def has_key(self, k):
+        # type: (Any) -> bool
+        return k in self.__dict__
+
+    def update(self, *args, **kwargs):
+        # type: (Any, Any) -> None
+        return self.__dict__.update(*args, **kwargs)
+
+    def keys(self):
+        # type: () -> list
+        return [k for k in self.__dict__ if not k.startswith("_")]
+
+    def values(self):
+        # type: () -> list
+        return [v for k, v in self.__dict__.items() if not k.startswith("_")]
+
+    def items(self):
+        # type: () -> list
+        return [(k, v) for k, v in self.__dict__.items() if not k.startswith("_")]
+
+    def get(self, key, default=None):
+        # type: (Any, Optional[Any]) -> Any
+        if key in self.__dict__:
+            return self.__dict__[key]
+        return default
+
+
+class AmqpAnnotatedMessage(object):
+    # pylint: disable=too-many-instance-attributes
+    """
+    The AMQP Annotated Message for advanced sending and receiving scenarios which allows you to
+    access to low-level AMQP message sections. There should be one and only one of either data_body, sequence_body
+    or value_body being set as the body of the AmqpAnnotatedMessage; if more than one body is set, `ValueError` will
+    be raised.
+    Please refer to the AMQP spec:
+    http://docs.oasis-open.org/amqp/core/v1.0/os/amqp-core-messaging-v1.0-os.html#section-message-format
+    for more information on the message format.
+
+    :keyword data_body: The body consists of one or more data sections and each section contains opaque binary data.
+    :paramtype data_body: Union[str, bytes, List[Union[str, bytes]]]
+    :keyword sequence_body: The body consists of one or more sequence sections and
+     each section contains an arbitrary number of structured data elements.
+    :paramtype sequence_body: List[Any]
+    :keyword value_body: The body consists of one amqp-value section and the section contains a single AMQP value.
+    :paramtype value_body: Any
+    :keyword header: The amqp message header.
+    :paramtype header: Optional[~azure.servicebus.amqp.AmqpMessageHeader]
+    :keyword footer: The amqp message footer.
+    :paramtype footer: Optional[dict]
+    :keyword properties: Properties to add to the amqp message.
+    :paramtype properties: Optional[~azure.servicebus.amqp.AmqpMessageProperties]
+    :keyword application_properties: Service specific application properties.
+    :paramtype application_properties: Optional[dict]
+    :keyword annotations: Service specific message annotations.
+    :paramtype annotations: Optional[dict]
+    :keyword delivery_annotations: Service specific delivery annotations.
+    :paramtype delivery_annotations: Optional[dict]
+
+    :raises: :class:`ValueError`
```

Comment:
I would remove this... raising builtin errors for invalid input doesn't really need documenting.

Reply:
removed!